Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob/Definitions/Result.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Frends.AzureBlobStorage.DeleteBlob
+{
+    /// <summary>
+    /// Result-class for DeleteBlob-task.
+    /// </summary>
+    public class Result
+    {
+        /// <summary>
+        /// Was the operation successful?
+        /// </summary>
+        public bool Success { get; set; }

```

Comment:
When success is false then do we have any error messages about it or something?..

Reply:
Not necessary, since that boolean actually indicates if the deletion command was successful. If the blob doesn't exist, then that value is false. But that is really not a case to throw an error, since the idea is to delete that blob anyway. If error occurs during deletion, the task will throw an error (The task is wrapped in try/catch). The deletion command `blob.DeleteIfExistsAsync` returns true if the blob exists and was deleted. Otherwise it returns false and no information about why it is false.